src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,31 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * The task with name and deadline.
+ */
+public class Deadline extends Task {
+    protected String by;
+    protected String date;
+
+
+    public Deadline(String name, String by) {
+        super(name);
+        this.by = by;
+        converter(by);
+    }
+
+    public void converter(String by) {
+        DateTimeFormatter inputFormatter = DateTimeFormatter.ofPattern("yyyy/MM/dd");
+        LocalDate inputDate = LocalDate.parse(by, inputFormatter);
+        DateTimeFormatter outputFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+        date = inputDate.format(outputFormatter);
+    }
```

Comment:
Not a coding violation. Perhaps you could consider a more descriptive name for clarity (convertDateFormat)?